kDriveCore/Data/UploadQueue/Queue/UploadQueue.swift

```diff
@@ -102,7 +102,8 @@ public final class UploadQueue: ParallelismHeuristicDelegate {
         }
 
         let status = ReachabilityListener.instance.currentStatus
-        return status == .offline || (status != .wifi && UserDefaults.shared.isWifiOnly)
+        let shouldBeSuspended = status == .offline || !(status == .wifi && UserDefaults.shared.isWifiOnly)
```

Comment:
I think It should be this instead:
```suggestion
        let shouldBeSuspended = status == .offline || (status != .wifi && UserDefaults.shared.isWifiOnly)
```